test.js

```diff
@@ -1,53 +1,53 @@
 import test from 'ava';
-import {patternToRegex} from './index.js';
+import {patternToRegex, globToRegex} from './index.js';
```

Comment:
Instead of making changes to this file, you can create an additional `test.globs.ts` and add the specific tests there.